src/components/BlockchainPanel.vue

```diff
@@ -21,9 +21,7 @@
           </div>
         </template>
         <template #tooltip>
-          Unlike in Bitcoin, (key)blocks do not directly include any transaction. Instead, a mined keyblock
-          determines a new leader who is then eligible to create new microblocks every 3 seconds which include
-          the transactions. On average, a new keyblock is mined every 3 minutes.
+          {{ keyblocksHints.keyblock }}
```

Comment:
I'd advise using a different hint name because this description does not explain what a keyblock exactly is. How about:
* keyblockLong
* keyblockFacts
* keyblockInComparison

I don't particularly like any of my proposals but I also think `.keyblock` should be reserved for keyblock details page title hint

Reply:
ok then, let's keep it this way